com.roundabout_host.Izvor.yaml

```diff
@@ -0,0 +1,70 @@
+app-id: com.roundabout_host.Izvor
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: izvor
+finish-args:
+  - --share=network
+  - --filesystem=home
+  - --filesystem=host
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=wayland
+  - --filesystem=xdg-data/izvor:create
+  - --filesystem=xdg-config/izvor:create
+environment:
```

Comment:
not sure what paragraph in the documentation said "environment" was valid. Can you tell so we fix the wrong documentation?